static/app/components/globalModal/components.tsx

```diff
@@ -1,6 +1,6 @@
 import styled from '@emotion/styled';
 
-import Button, {ButtonPropsWithAriaLabel} from 'sentry/components/button';
```

Comment:
Didn't need to import this. it was the only place importing this exported interface.